pkg/addon/render.go

```diff
@@ -133,6 +133,10 @@ func (a addonCueTemplateRender) renderApp() (*v1beta1.Application, []*unstructur
 	if err != nil {
 		return nil, nil, errors.Wrap(err, "load app template with CUE files")
 	}
+	if v.Error() != nil {
+		return nil, nil, errors.Wrap(v.Error(), "load app template with CUE files")
+	}
```

Comment:
why this check necessary here?

Reply:
The new version of cue wraps all errors into `value.error` instead of returning the error.